src/Rules.Framework.WebUI/Handlers/GetPriorityCriteriaHandler.cs

```diff
@@ -0,0 +1,36 @@
+namespace Rules.Framework.WebUI.Handlers
+{
+    using System;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Rules.Framework.Generics;
+
+    internal sealed class GetPriorityCriteriaHandler : WebUIRequestHandlerBase
+    {
+        private static readonly string[] resourcePath = new[] { "/{0}/api/v1/configurations" };
+
+        private readonly IGenericRulesEngine rulesEngine;
+
+        public GetPriorityCriteriaHandler(IGenericRulesEngine rulesEngine, WebUIOptions webUIOptions) : base(resourcePath, webUIOptions)
+        {
+            this.rulesEngine = rulesEngine;
+        }
+
+        protected override HttpMethod HttpMethod => HttpMethod.GET;
+
+        protected override Task HandleRequestAsync(HttpRequest httpRequest, HttpResponse httpResponse, RequestDelegate next)
+        {
+            try
+            {
+                var priorityCriteria = this.rulesEngine.GetPriorityCriteria();
+
+                return this.WriteResponseAsync(httpResponse, priorityCriteria.ToString(), (int)HttpStatusCode.OK);
```

Comment:
This way you only expose the priority criteria value directly. To keep this simple and extensible to add more configurations in the future, can we please create a dictionary, add the priority criteria there and write that dictionary to response?

Reply:
Done